src/Raven.Client/Server/Commands/GetTopologyCommand.cs

```diff
@@ -14,7 +14,7 @@ public GetTopologyCommand()
 
         public override HttpRequestMessage CreateRequest(ServerNode node, out string url)
         {
-            url = $"{node.Url}/databases/{node.Database}/topology?url={node.Url}";
+            url = $"{node.Url}/admin/databases/topology?&name={node.Database}";
```

Comment:
This is not an admin operation. We are better doing something like: `/topology?name=` instead.
And we need to consider the url there, it is there for a reason, since we might have different endpoints.